### ISSUE TITLE
Release Core-1.4.1

### ISSUE THREAD

**Issue by craigh** [zikula/core#2641]
Please test [the most recent build or Core-1.4.1](http://ci.zikula.org/job/Zikula_Core-1.4.1/) and decide if it should become the next official release. Anyone may participate in the testing process.

Testing guidelines can be found in [Release Testing Guidelines](https://github.com/zikula/core/wiki/Release-Testing-Guidelines)

Contrary to previous releases, this release will not require a voting process, but will rather be a "time-based" release. In other words, barring any issue that rises to the 'blocker' level, **the release will occur on 20 Nov 2015**.

Please report issues in a separate ticket. Minor and major bugs may not block a release, but will be scheduled in a future version.

**Comment by craigh** [comment]
currently blocked by #2642 and translation in general @Kaik

**Comment by craigh** [comment]
#2642 is now fixed and no longer blocking release.

**Comment by craigh** [comment]
now blocked by #2649

**Comment by craigh** [comment]
#2649 is now fixed and no longer blocking release.

**Comment by craigh** [comment]
released.

**Comment by rallek** [comment]
great job! Thanks to the team!
